.pipelines/cni/pipeline.yaml

```diff
@@ -359,6 +370,40 @@ stages:
       vmSize: ${VM_SIZE_CILIUM}
       dualstackVersion: ${CILIUM_DUALSTACK_VERSION}
       cni: "cilium_dualstack"
+  
+  - template: cilium/cilium-overlay-load-test-template.yaml
+    parameters:
+      name: cilium_ds_arm
+      clusterType: dualstack-byocni-nokubeproxy-up
+      clusterName: "cil-ds-arm"
+      nodeCount: ${NODE_COUNT_CILIUM}
+      vmSize: Standard_D8ps_v5
+      arch: arm64
+      dualstackVersion: ${CILIUM_DUALSTACK_VERSION}
+      cni: "cilium_dualstack"
+
+  - template: cilium/cilium-overlay-load-test-template.yaml
+    parameters:
+      name: cilium_ds_mariner
+      clusterType: dualstack-byocni-nokubeproxy-up
+      clusterName: "cil-ds-mar"
+      nodeCount: ${NODE_COUNT_CILIUM}
+      vmSize: ${VM_SIZE_CILIUM}
+      arch: amd64
+      osSKU: AzureLinux
+      dualstackVersion: ${CILIUM_DUALSTACK_VERSION}
+      cni: "cilium_dualstack"
+  
+  - template: cilium/cilium-overlay-load-test-template.yaml
+    parameters:
+      name: cilium_ds_rdma
+      clusterType: dualstack-byocni-nokubeproxy-up
+      clusterName: "cil-ds-rdma"
+      nodeCount: ${NODE_COUNT_CILIUM}
+      vmSize: Standard_HC44-16rs

```

Comment:
We might be getting close to our RDMA quota limit... Need to keep an eye on cluster creation after this is merged.